internal/datacoord/util.go

```diff
@@ -148,7 +148,7 @@ func getCompactedSegmentSize(s *datapb.CompactionSegment) int64 {
 	if s != nil {
 		for _, binlogs := range s.GetInsertLogs() {
```

Comment:
1. should we use memory size or file size?
2. why delta use Log size but insert use file size?
3. GetDeltalogs  is duplicated

Reply:
will use memory size